js/src/common/components/ErrorPage.js

```diff
@@ -0,0 +1,40 @@
+import Link from './Link';
+import Page from './Page';
+
+export default class ErrorPage extends Page {
+  oninit(vnode) {
+    super.oninit(vnode);
+
+    this.title = app.translator.trans(`core.forum.error.${this.attrs.errorCode}_title`);
+  }
+
+  oncreate(vnode) {
+    super.oncreate(vnode);
+
+    app.setTitle(this.title);
+  }
+
+  view() {
+    const links = {
+      404: '/',
+    };
+
+    const link = links[this.attrs.errorCode];
+
+    return (
+      <div className="ErrorPage">
+        <div className="container">
+          <h2>{this.title}</h2>
+          <p>{app.translator.trans(`core.forum.error.${this.attrs.errorCode}_text`)}</p>
```

Comment:
@frontend folks, any ideas for making this look... not horrible?

Reply:
My first idea would be bigger text. 13px is fine for forum posts and dense pages, but an error page needs to be padded out in some form, I think.

We don't want to do too much theming as that should be up to forum maintainers, though.